app/src/ui/app.tsx

```diff
@@ -1284,12 +1284,14 @@ export class App extends React.Component<IAppProps, IAppState> {
           />
         )
       case PopupType.About:
+        const version = __DEV__ ? __SHA__.substr(0, 10) : getVersion()
```

Comment:
The `.substr` here is a rough approximation of `git rev-parse --short HEAD` (currently returning 9 characters):

```
$ git rev-parse --short HEAD
7159d344a
```

I _almost_ went with wiring up a Git operation to do this for maximum correctness, but it turns out the way we get `__SHA__` doesn't invoke Git at all:

https://github.com/desktop/desktop/blob/7159d344a026aae6aa8f85c89b130edf6acebbbe/app/git-info.ts#L49-L77